daemon/algod/api/server/v2/utils.go

```diff
@@ -626,6 +627,49 @@ func convertSimulationRequest(request PreEncodedSimulateRequest) simulation.Requ
 	}
 }
 
+func convertLedgerStateDelta(sd ledgercore.StateDelta) LedgerStateDeltaJSONSerializable {
+	serializableTxleases := make([]TxleaseJSONSerializable, 0, len(sd.Txleases))
+	for k, v := range sd.Txleases {
+		serializableTxleases = append(serializableTxleases, TxleaseJSONSerializable{
+			Sender:     k.Sender,
+			Lease:      k.Lease,
+			Expiration: v,
+		})
+	}
+	// TODO: sort for deterministic response
```

Comment:
What would it be sorted based on? Maybe each `Txlease` is hashed and the digests are compared? Or they could be formatted with `%+v` and then ordered based on the resulting strings? The latter would probably be faster.

Reply:
Added in 361d1c6